packages/mobile/src/account/SyncBankAccountScreen.tsx

```diff
@@ -40,7 +40,7 @@ const SyncBankAccountScreen = ({ route }: Props) => {
 
       navigate(Screens.BankAccounts, { newPublicToken: publicToken })
     } catch {
-      // TODO(wallet#1447): handle errors from IHL
+      navigate(Screens.LinkBankAccountErrorScreen)
```

Comment:
Can you instead do `catch (error) {` and pass the error along so it can be logged?